.rubocop.yml

```diff
@@ -1,6 +1,7 @@
 ---
 require:
 - rubocop-rspec
```

Comment:
if you want, you can also use the rubocop config we ship in voxpupuli-test. That's designed for modules that are compatible with Puppet 7 and 8 (and I hope to move to 8 soon).

```yaml
---
inherit_gem:
  voxpupuli-test: rubocop.yml
```

Reply:
The vox config intentionally wasn't used because I disagree with with removing trailing commas, the `.` on line wrapped method calls being trailing instead of preceding, etc.

Let me see how bad it is to use it as a base with overrides.

Reply:
I also want the trailing commas back, but reintroducing at vox pupuli causes a lot of work. I hope I can combine that with the drop of ruby 2.7 soonish.

Reply:
It wasn't as bad as I thought it was going to be.